packages/core/src/components/forms/numericInput.tsx

```diff
@@ -236,8 +246,11 @@ export class NumericInput extends AbstractComponent<HTMLInputProps & INumericInp
             const decrementButton = this.renderButton(
                 NumericInput.DECREMENT_KEY, NumericInput.DECREMENT_ICON_NAME, this.handleDecrementButtonClick);
 
+            const buttonGroupClasses = classNames(Classes.BUTTON_GROUP, Classes.VERTICAL, Classes.FIXED, {
+                [Classes.LARGE]: large,
```

Comment:
actually i don't think this does anything. there are no styles for `.pt-button-group.pt-large`.

the selector is `.pt-button.pt-large, .pt-large .pt-button`.